### ISSUE TITLE
Bower package name not consistent

### ISSUE THREAD

**Issue by adam-lynch** [medialize/URI.js#195]
In the `bower.json` it's `URIjs` but the readme says to run `bower install uri.js`. Both work (not sure how?).

**Comment by rodneyrehm** [comment]
yes, this stems from a time where capitals were still allowed in bower/npm. We'll fix this with version 2 (whenever we start tackling it).

**Comment by rodneyrehm** [comment]
I have just renamed the package to `urijs` in npm (was `URIjs`) and bower (was `uri.js`).

new on npm: [urijs](https://www.npmjs.com/package/urijs)
old on npm [URIjs](https://www.npmjs.com/package/URIjs)

```
➜  bower search uri.js
Search results:

    uri.js git://github.com/medialize/URI.js.git
    URIjs git://github.com/medialize/URI.js.git
    urijs git://github.com/medialize/URI.js.git (<---)
```

**Comment by ooxi** [comment]
Does this mean old references do not work any longer in bower and npm or is there some kind of automatic redirection?

**Comment by rodneyrehm** [comment]
The old reference ("URIjs") will still work - up to the the version 1.16.1

**Comment by ooxi** [comment]
Hm yes that's what I figured. But now automatic checks for newer versions won't work and users might not be informed about a newer URIjs version.

How about publishing a final `1.16.2-URIjs-renamed-to-urijs` version?

**Comment by adam-lynch** [comment]
You can deprecate versions with a message (I'm pretty sure you can deprecate all versions too)

**Comment by rodneyrehm** [comment]
> Hm yes that's what I figured. But now automatic checks for newer versions won't work and users might not be informed about a newer URIjs version.

True. Furthermore most of the reasons that made me rename the package are void. In a little while someone will be joining the npm team who might make this "meta project maintenance" stuff a priority topic. we'll see what they'll come up with to get me/us out of this mess.

> How about publishing a final 1.16.2-URIjs-renamed-to-urijs version?

I thought [that's what I did](https://www.npmjs.com/package/URIjs).

> You can deprecate versions with a message (I'm pretty sure you can deprecate all versions too)

just ran 

```
npm deprecate URIjs@"< 1.16.2" 'package renamed to "urijs" (lower-case), please update accordingly'
```

---

The issue that finally got me to rename the package was that [URIjs](https://www.npmjs.com/package/URIjs) was not showing any download numbers, and dependent packages anymore. Because npm moved to lower-case package names a while ago, and various issues with that have not been treated with any priority (at least it did not feel that way), I decided to simply side-step this and any future issue by renaming the package. 

As of *now* the stats are back (except for historic download numbers, but whatever). Making the entire renaming thing kinda pointless. I'm going to crawl back under the rock I came from - have a good day ;)
